src/app/components/file-browser/terra-frontend-storage.service.ts

```diff
@@ -96,10 +96,13 @@ export class TerraFrontendStorageService extends TerraBaseMetadataStorageService
         }
 
         let uploadItems:Array<TerraUploadItem> = [];
-        for(let i = 0; i < files.length; i++)
+
+        /* tslint:disable:prefer-for-of */
```

Comment:
Warum nicht umbauen?

Reply:
Das hier ist ein ganz spezieller Sonderfall. 
Es kann nicht per for of oder forEach über files iteriert werden, hat mit dem Aufbau des Objectes zu tun.
Die [] Klammern wurden aber überschrieben für das Objekt deswegen kann darauf zugegriffen werden.
Die einzige andere Option wäre ein tiefergehender Umbau.

Reply:
ah ok :)